### PR TITLE
VIH-7946 video web url fix

### PR DESCRIPTION
**Pull request by lowflying**
### JIRA link (if applicable) ###
N/A

### Change description ###

added logic for the VhServices:VideoWebUrl app setting so it plays nice with the vanity urls in prod/preprod

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: azure-pipelines.release.yml [hunk 1]
```diff
@@ -295,7 +295,12 @@ stages:
           - name: VhServices:VideoApiUrl
             value: $(video_api_url)
           - name: VhServices:VideoWebUrl
-            value: $(video_web_url)
+            ${{ if containsValue('preview', parameters.environment) }}:
```
Comment (robertbennett1998) [inline azure-pipelines.release.yml:298]: Should this be preprod?

File: azure-pipelines.release.yml [hunk 2]
```diff
@@ -295,7 +295,12 @@ stages:
...
+              value: $(video_web_url_preprod)
+            ${{ if containsValue('prod', parameters.environment) }}:
+              value: $(video_web_url_prod)                      
+            ${{ if not(or(containsValue('preprod', parameters.environment), containsValue('preprod', parameters.environment))) }}:
```
Comment (robertbennett1998) [inline azure-pipelines.release.yml:302]: Should this be if 
`not(or(containsValue('preprod', parameters.environment), containsValue('prod', parameters.environment))) ?`

**Review by robertbennett1998 — CHANGES_REQUESTED** [review]

File: azure-pipelines.release.yml (hunk 1 above)
Comment (lowflying) [reply azure-pipelines.release.yml:298]: yes indeed it should

**Review by vialde — APPROVED** [review]

**Review by robertbennett1998 — APPROVED** [review]
LGTM

**Review by vialde — APPROVED** [review]

**Pull request merged**